lib.sh

```diff
@@ -230,14 +251,14 @@ domain_check_200() {
     install_if_not dnsutils
 
     # Try to resolve the domain with nslookup using $DNS as resolver
-    if nslookup "${1}" $DNS1 >/dev/null 2>&1
+    if nslookup "${1}" 9.9.9.9 >/dev/null 2>&1
```

Comment:
Why not use the variable here?

Reply:
Added INTERNET_DNS variable at the top of lib.sh to distinguish it from DNS1 that might be an internal DNS server. In the places where DNS1 was used in lib.sh, an external DNS server is needed, so we have to use a Quad9 default as we can't say what the changed DNS1 might be.